backend/src/main/java/gov/cdc/usds/simplereport/service/sms/TextMessageStatusService.java

```diff
@@ -74,4 +86,44 @@ private List<String> getQueryStringKeys(String queryString) {
           .collect(Collectors.toList());
     }
   }
+
+  private String getNumberByMessageId(String messageId, String twilioNumber) {
```

Comment:
@emmastephenson after our Twilio troubleshooting session I realized that the number sent in the status request is masked (as you pointed out). So I had to add a method to retrieve the phone number using the messageId like once we discussed. Please let me know if any questions.